feature/qr-test-doc (#2304)
mercadopago/devsite-docs: guides/qr-code/qr-goto-production.pt.md

# Entrar em produção

> Quando sua integração estiver pronta e você quiser começar a receber pagamentos, ative suas [credenciais de produção](/developers/pt/docs/qr-code/additional-content/your-integrations/credentials) e substitua as de teste. 

Reviewer: ```suggestion

Ao finalizar os testes e garantir o correto funcionamento da integração, será necessário colocá-la em produção, o que permitirá o recebimento de pagamentos reais. Para isso, ative suas [credenciais de produção](/developers/pt/docs/qr-code/additional-content/your-integrations/credentials) e substitua as de teste. 
```

> Além disso, recomendamos que você cumpra as seguintes considerações adicionais.

Reviewer: ```suggestion
Feito isso, siga as etapas abaixo e garanta que cada uma delas seja cumprida. Isso garantirá que a saída à produção seja feita com êxito.
```

## Homologação 

> Para oferecer a melhor experiência tanto para o vendedor quanto para o comprador, é importante que você possa **validar a qualidade da sua integração** de acordo com os padrões do Mercado Pago antes de ir para a produção. 

Reviewer: ```suggestion
Para oferecer a melhor experiência tanto para o vendedor quanto para o comprador, é importante **validar a qualidade da sua integração** de acordo com os padrões do Mercado Pago antes de ir para a produção. 
```

> O processo de homologação permitirá que você certifique que sua integração atenda aos requisitos de qualidade necessários e, caso contrário, irá sugerir os ajustes que você precisa fazer para otimizar seu desenvolvimento ao receber pagamentos reais. 

Reviewer: ```suggestion
O processo de homologação possibilita a certificação de que a integração cumpre com os requisitos de qualidade necessários. Caso não atenda, serão sugeridos ajustes necessários para otimizar o desenvolvimento antes do recebimento de pagamentos reais.
```

> Para medir a qualidade da sua integração com o Código QR, siga os passos abaixo.

Reviewer: ```suggestion
Para medir a qualidade da sua integração com o Código QR, siga as etapas descritas abaixo.
```

> WARNING
>
> Importante
> 

> > O processo de homologação deve ser realizado para o aplicativo que você criou com suas **credenciais de produção**.

Reviewer: ```suggestion
> O processo de homologação deve ser realizado para o aplicativo que criado com as **credenciais de produção**.
```

1. Acesse [Suas integrações](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/panel/app) no painel superior direito do Devsite. 

> 2. Em seguida, clique no aplicativo que você criou e selecione **Avaliar a qualidade** para acessar a ferramenta onde você pode medir a qualidade da sua integração.

Reviewer: ```suggestion
2. Em seguida, clique no aplicativo criado e selecione **Avaliar a qualidade** para acessar a ferramenta na qual você pode medir a qualidade da sua integração.
```

![Detalhes da aplicação no Painel do Desenvolvedor](/images/qr/homologacion-qr-pt.png)

> 3. Siga as instruções fornecidas pela ferramenta de homologação para realizar o processo. Lembre-se de que as ações indicadas como **necessárias** devem ser concluídas para acumular pontos que irão melhorar a qualidade da sua integração, enquanto as indicadas como **boas práticas** são recomendadas, mas não afetarão a pontuação.

Reviewer: ```suggestion
3. Siga as instruções fornecidas pela ferramenta de homologação para realizar o processo. Tenha em mente que as ações indicadas como **necessárias** devem ser concluídas para acumular pontos que irão melhorar a qualidade da sua integração, enquanto as indicadas como **boas práticas** são recomendadas, mas não afetarão a pontuação.
```

> NOTE
>
> Nota
> 
> Se você faz parte da nossa Carteira Assessorada, entre em contato com a equipe de Integrações para realizar a homologação.

## Certificado SSL
Para que sua integração seja segura e proteja os dados das transações, é necessário ter um **certificado SSL**. 

> Com este processo, você pode garantir a segurança dos dados de seus clientes, cumprir os regulamentos ou disposições legais de cada país e conseguir a melhor experiência de compra para suas vendas.

Reviewer: ```suggestion
Com isto, é possível garantir a segurança dos dados de seus clientes, cumprir os regulamentos ou disposições legais de cada país e conseguir a melhor experiência de compra para suas vendas.
```

Apesar de não ser obrigatório durante a fase de testes, o certificado é obrigatório para iniciar em produção.

## Relatórios
Os [relatórios do Mercado Pago](/developers/pt/docs/qr-code/additional-content/reports/introduction) fornecem informações financeiras para acompanhar as movimentações da sua conta, como saldo disponível, transações e liquidez. Isso facilita a conciliação das vendas e outras operações com seus sistemas de gestão internos.

> Recomendamos que você utilize os relatórios para melhorar a gestão financeira da sua empresa assim que iniciar a produção.

Reviewer: ```suggestion
Recomendamos utilizar os relatórios para melhorar a gestão financeira da sua empresa assim que iniciar a produção.
```

Reviewer: @pcanosa - Pili, esse "iniciar a produção" seria relacionado à transição do ambiente de teste para o de produção? Se for isso, precisamos deixar um pouquinho mais claro. 

Quem ler pode pensar: mas produção de quê?
